pyproject.toml

```diff
@@ -15,3 +15,11 @@ no_implicit_optional = true
 warn_redundant_casts = true
 warn_unused_ignores = true
 warn_return_any = true
+plugins = [
+  "pydantic.mypy"
+]
+
+[tool.pydantic-mypy]
+init_forbid_extra = true
+init_typed = true
+warn_required_dynamic_aliases = true
```

Comment:
brrp, not a [posix text file](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline/729795#729795).